fem/bilininteg_convection_pa.cpp

```diff
@@ -783,7 +783,7 @@ void ConvectionIntegrator::AssemblePA(const FiniteElementSpace &fes)
    }
    const int dims = el.GetDim();
    const int symmDims = dims;
-   const int nq = ir->GetNPoints();
+   nq = ir->GetNPoints();
```

Comment:
```suggestion
```

Reply:
Again, this may not be set anyplace else. Also, `nq` will depend on the element type so it may change as we loop over elements (but such meshes don't seem to be supported by PA).

Do we need `ne` and `nq` as member data in this class? Might it be better to just use these local variables?

Reply:
I agree with you, my mistake. `ne` is reused in the `Mult`, `nq` though could only be local as I don't think it is used anywhere else.

Reply:
You can keep it the way you prefer though, I don't have strong feelings about it.